selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOrganizationTest.java

```diff
@@ -51,19 +50,16 @@
   private OrganizationDto parentOrganization;
   private OrganizationDto childOrganization;
 
-  @Inject private TestUser defaultTestUser;
   @Inject private OrganizationListPage organizationListPage;
   @Inject private OrganizationPage organizationPage;
   @Inject private NavigationBar navigationBar;
   @Inject private CheMultiuserAdminDashboard dashboard;
+  @Inject private TestUser testUser;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
-  @Inject private TestUser testUser;
-  @Inject private AdminTestUser adminTestUser;
```

Comment:
Looks strange - **AdminOrganizationTest** without **AdminTestUser**. It seems it doesn't test Admin dashboard at all. Do you have an issue to fix this moment?

Reply:
Tests from organization package will be reworked by https://github.com/eclipse/che/issues/7121.